modules/install/pages/install-production-deployment.adoc

```diff
@@ -58,3 +58,16 @@ However, you might find a need to have a smaller deployment for test and develop
 
 xref:deployment-considerations-lt-3nodes.adoc[About Deploying Clusters With Less Than Three Nodes]
 |===
+
+== General Guidelines
+
+Use RAID 1+0 (or RAID 1) if Couchbase bucket replication is 1 or lower:: If bucket replicas are set to 2 or higher, the administrator might rely on database-level replication for data durability.
```

Comment:
Not sure what the origin of this recommendation is. While RAID 1 (mirroring) is certainly a reasonable choice, other RAID levels such as 5 or 6 are perfectly valid choices.

Personally I think we should shy away from specific RAID levels here, instead saying something along the lines of:

> "To reduce the likelihood of disk failure affecting data integrity, consider using a disk configuration such as RAID with mirroring (RAID 1, RAID10) or parity (RAID 5, 6) to ensure that a single disk failure does not loose data.

You can keep the second sentence ("If bucket replicas are 2 or higher...").

Reply:
Updated.